settings.json

```diff
@@ -44,7 +48,25 @@
     "dbpedia":     "http://dbpedia.org/resource/",
     "dbpedia-owl": "http://dbpedia.org/ontology/",
     "dbpprop":     "http://dbpedia.org/property/",
-    "schema":      "http://schema.org/"
+    "schema":      "http://schema.org/",
+    "wikibase":    "http://wikiba.se/ontology#",
+    "wdata":       "https://www.wikidata.org/wiki/Special:EntityData/",
+    "wd":          "http://www.wikidata.org/entity/",
+    "wds":         "http://www.wikidata.org/entity/statement/",
+    "wdref":       "http://www.wikidata.org/reference/",
+    "wdv":         "http://www.wikidata.org/value/",
+    "wdt":         "http://www.wikidata.org/prop/direct/",
+    "p":           "http://www.wikidata.org/prop/",
+    "ps":          "http://www.wikidata.org/prop/statement/",
+    "psv":         "http://www.wikidata.org/prop/statement/value/",
+    "psn":         "http://www.wikidata.org/prop/statement/value-normalized/",
+    "pq":          "http://www.wikidata.org/prop/qualifier/",
+    "pqv":         "http://www.wikidata.org/prop/qualifier/value/",
+    "pqn":         "http://www.wikidata.org/prop/qualifier/value-normalized/",
+    "pr":          "http://www.wikidata.org/prop/reference/",
+    "prv":         "http://www.wikidata.org/prop/reference/value/",
+    "prn":         "http://www.wikidata.org/prop/reference/value-normalized/",
+    "wdno":        "http://www.wikidata.org/prop/novalue/"
```

Comment:
Can we avoid this? Maybe with explicit prefixes in that query? Because every `CONSTRUCT` query would list all of these.

Reply:
Ah, I didn't think of that, will remove the non-essential ones.